core/diagnostics/DiagnosticScenarios/readme.md

```diff
@@ -16,7 +16,7 @@ The sample debug target is a simple `webapi` application. The sample triggers un
 
 After downloading the source, you can easily run the webapi using:
 
-```console
+```dotnetcli
 dotnet build
 dotnet run
 ```
```

Comment:
Thanks @Youssef1313! The links on lines 30, 36, 42, 48 could use a code block with the http identifier as well. Then people should get a copy link like [here](https://docs.microsoft.com/en-us/dotnet/framework/data/wcf/querying-the-data-service-wcf-data-services#linq-queries)

Reply:
@mairaw done.